### PR TITLE
prep 0.4.0

### PR DESCRIPTION
**Pull request by jburel**
* Use the idr channel
* remove dependencies
* remove dev
* fix test

cc @manics

### CODE REVIEW

File: idr/externalDBs.py [hunk 1]
```diff
@@ -78,7 +78,8 @@ def get_ensembleid(gene):
     if results.empty:
         return ensembleid
 
-    for i in results.ix[:, 0]:
+    print(res.text)
```
Comment (manics) [inline idr/externalDBs.py]:
```suggestion
```
Was this for local debugging?

Comment (jburel) [reply idr/externalDBs.py]: yes thanks for spotting it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+0.4.0 (2020-03-17)
+------------------
+
+Features:
+
+- Add support for websockets [#20](https://github.com/IDR/idr-py/pull/20)
+
```
Comment (manics) [inline CHANGELOG.md]: Do you think we should squash the `0.4.0.dev*` entries into a single `0.4.0` entry for the final release?

Comment (jburel) [reply CHANGELOG.md]: Can do

**Comment by manics** [comment]
👍

**Pull request merged**